docker-compose.yml

```diff
@@ -21,4 +21,4 @@ services:
       - reposado
 
 volumes:
-  reposado:
+  reposado: {}
```

Comment:
Elaborate on this?

Reply:
I don't know if this has any consequence; idiosyncrasy.